Controllers/reviews.controllers.js

```diff
@@ -9,4 +9,23 @@ res.status(200).send({ review })
     
 next(err)
 })
+}
+
+exports.patchReview = (req, res, next) => {
+    const review_id = req.params.review_id
+    let alterVotes = req.body.inc_votes  
+
+    fetchReview(review_id).then((review) => {
+        let votes = review[0].votes
+        let newVotes = votes += alterVotes
+        
```

Comment:
Hey Chris!

How come you are fetching the review here before updating? I would say all the behaviour required can be handled in the one updateReview model function!